docker-compose.yml

```diff
@@ -4,34 +4,104 @@ version: "3.7"
 # This Docker composition file is used to build and test the container
 
 secrets:
-  quote_txt:
-    file: ./src/secrets/quote.txt
+  admiral_yml:
+    file: ./src/secrets/admiral.yml
+  mongo_root_passwd_txt:
+    file: ./src/secrets/mongo-root-passwd.txt
+  redis_conf:
+    file: ./src/secrets/redis.conf
+  sslmate_api_key_txt:
+    file: ./src/secrets/sslmate-api-key.txt
+
+x-admiral-template: &admiral-template
+  build:
+    context: .
+    dockerfile: Dockerfile
+  deploy:
+    mode: replicated
+    replicas: 5
+  environment:
+    ADMIRAL_CONFIG_FILE: "/run/secrets/admiral.yml"
+    ADMIRAL_CONFIG_SECTION: dev-mode
+    ADMIRAL_WORKER_NAME: dev
+  image: cisagov/admiral
+  init: true
+  secrets:
+    - source: admiral_yml
+      target: admiral.yml
+    - source: sslmate_api_key_txt
+      target: sslmate-api-key.txt
+  # map for development only, comment out otherwise
```

Comment:
Can this comment be clarified a bit more?  It's unclear to me what it refers to and when exactly someone would uncomment it.

Reply:
Would it make more sense to remove this block? I suppose @felddy intended this for development of the Python source code, so we shouldn’t need it in this repository.

Reply:
If it's no longer useful, then I completely agree that it should be removed.